packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
Is the `.hjson` intentional? What does it mean??

Reply:
It's just a json format that allows me to add comments which makes it really easy to play around with the report transforms. Won't be in the final product 👍 

@kael89 also recommended jsonc for this, which I might switch to as prettier doesn't accept hjson so it doesn't auto format the file 😢

Reply:
👍 for `.jsonc`, also it's fine if we commit those as templates/documentation in cases it makes sense, eg [dashboardReports.example.jsonc](https://github.com/beyondessential/tupaia/blob/dev/packages/web-frontend/cypress/config/dashboardReports.example.jsonc)